crates/web_ui/src/user.rs

```diff
@@ -175,8 +178,8 @@ pub async fn logout(
         None => return Ok(jar), // Already logged out as no cookie can be found
     };
 
-    jar = jar.remove(Cookie::named(COOKIE_SESSION_ID));
-    jar = jar.remove(Cookie::build(COOKIE_SESSION_USER, "").path("/").finish());
+    jar = jar.remove(Cookie::build(COOKIE_SESSION_ID));
```

Comment:
```suggestion
    jar = jar.remove(COOKIE_SESSION_ID);
```

This should be the same, right?